lib/board.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+require_relative 'ui.rb'
+# Board class
+class Board
+  attr_reader :board
+
+  def initialize
+    @board = [1, 2, 3, 4, 5, 6, 7, 8, 9]
+    @winner_points = [0, 1, 2, 3, 4, 5, 6, 7, 8, 0, 3, 6,
+                      1, 4, 7, 2, 5, 8, 0, 4, 8, 6, 4, 2]
+  end
+
+  def update_board(num, char)
+    @board[num - 1] = char
+  end
+
+  def not_full?
```

Comment:
This method is already doing an awesome job. However, we can probably change it to just check whether the board is full (instead of checking if it is **not** full):

```ruby
def is_full?
 @board.all? String
end
```

Then of course we have to change the invocation in `Game` where we needed this method.